component/client-handler/pom.xml

```diff
@@ -166,7 +166,7 @@
                                         <limit implementation="org.jacoco.report.check.Limit">
```

Comment:
Don't we bump up the version ? This is not a backward compatible version of this authenticator. Hence shall we do it ? Also we need to have a branch for the old series of versions.